docs/en/short-track/course-overview.md

```diff
@@ -0,0 +1,29 @@
+## 🎯 Goals of the Qualifying Stage
+
+The Qualifying Stage is designed for students who are motivated to build a career in IT and are interested in pathways that may lead to opportunities at **EPAM**. Its purpose is to confirm your technical foundation (Markdown, HTML/CSS, TypeScript/JavaScript/Git) and readiness for advanced training. Successful participants will move on to the **Short Track (\~3.5 months)**, which provides React or Angular practice, team project experience, and prepares you to be considered for recommendations to **EPAM and other companies**. More than **1000 RS School graduates have started their careers at EPAM**, and around **24% of RS School mentors are EPAM employees**.
+
+## Learning Pathway
+
+### 1. Qualifying Stage Path
+
+- Start: **September 22, 2025**.
+- Duration: **8 weeks** (5 weeks of assignments, \~1 week for interview preparation, then interviews).
```

Comment:
`Duration: **8 weeks** (7 weeks of assignments, ~1 week for interview preparation, then interviews).`